nexus/src/app/instance_platform/cpu_platform.rs

```diff
@@ -846,6 +1078,76 @@ mod test {
         cpuid_leaf!(0x80000021, 0x00000045, 0x00000000, 0x00000000, 0x00000000),
     ];
 
+    // This CPUID leaf blob is some small tweaks on top of the "ideal Milan",
+    // maintaining some details that are disabled due to needed bhyve support
+    // and including Turin-specific features as supported and relevant to
+    // guests.
+    const TURIN_V1_CPUID: [CpuidEntry; 25] = [
+        cpuid_leaf!(0x0, 0x0000000D, 0x68747541, 0x444D4163, 0x69746E65),
+        cpuid_leaf!(0x1, 0x00B00F21, 0x00000800, 0xF6D83203, 0x078BFBFF),
+        cpuid_leaf!(0x5, 0x00000000, 0x00000000, 0x00000000, 0x00000000),
+        cpuid_leaf!(0x6, 0x00000004, 0x00000000, 0x00000000, 0x00000000),
+        cpuid_subleaf!(
+            0x7, 0x0, 0x00000001, 0xF1BB03A9, 0x18005F42, 0x00000110
+        ),
+        cpuid_subleaf!(
+            0x7, 0x1, 0x00000030, 0x00000000, 0x00000000, 0x00000000
+        ),
+        cpuid_subleaf!(
+            0xD, 0x0, 0x000000E7, 0x00000980, 0x00000980, 0x00000000
+        ),
+        cpuid_subleaf!(
+            0xD, 0x1, 0x00000007, 0x00000980, 0x00000000, 0x00000000
+        ),
+        cpuid_subleaf!(
+            0xD, 0x2, 0x00000100, 0x00000240, 0x00000000, 0x00000000
+        ),
+        /*
+         * subleaves 3 and 4 are all-zero
+         */
+        cpuid_subleaf!(
+            0xD, 0x5, 0x00000040, 0x00000340, 0x00000000, 0x00000000
+        ),
+        cpuid_subleaf!(
+            0xD, 0x6, 0x00000200, 0x00000380, 0x00000000, 0x00000000
+        ),
+        cpuid_subleaf!(
+            0xD, 0x7, 0x00000400, 0x00000580, 0x00000000, 0x00000000
+        ),
+        cpuid_leaf!(0x80000000, 0x80000021, 0x68747541, 0x444D4163, 0x69746E65),
+        cpuid_leaf!(0x80000001, 0x00B00F21, 0x40000000, 0x440001F1, 0x25D3FBFF),
+        cpuid_leaf!(0x80000002, 0x6469784F, 0x69562065, 0x61757472, 0x7554206C),
+        cpuid_leaf!(0x80000003, 0x2D6E6972, 0x656B696C, 0x6F725020, 0x73736563),
+        cpuid_leaf!(0x80000004, 0x2020726F, 0x20202020, 0x20202020, 0x00202020),
+        cpuid_leaf!(0x80000007, 0x00000000, 0x00000000, 0x00000000, 0x00000100),
+        cpuid_leaf!(0x80000008, 0x00003030, 0x20000005, 0x00000000, 0x00000000),
```

Comment:
Just confirming, we cap %eax at 0x30/0t48 because we don't support virtualizing 5 level paging, right?